### DISCUSSION TITLE
Player`s location cannot be changed while using MCC

### DISCUSSION THREAD

**Discussion by Sagittaro** [ORelio/Minecraft-Console-Client#2319] [Q&A] [answered]
https://user-images.githubusercontent.com/51532982/199630436-bfb54b4e-1c2d-491f-a7e8-d2d2b553ca15.mp4

In this video, the leather suit player is using MCC to afk. While afking, other players cannot push his hitbox, even break the block under him, he cannot fall down but floating just like a hacker. Admin receive the report and ban him. So What is the cause?

**Comment by milutinke** [comment, marked as answer]
Hitboxes and physics are not implemented.
As for gravity, turn it on in the configuration file or use `/move gravity on`, but you have to have Terrain and Movements enabled.
